packages/ui-reactive/src/Unlocking.tsx

```diff
@@ -39,18 +39,17 @@ export class UnlockingDisplay extends React.PureComponent<Props> {
   private groupByEra (list: UnlockChunk[]) {
 
     return (
-      list.reduce((map: Map<string, Balance>, { era, value }) => {
-        const key = era.toString();
-
-        if (!map.has(key)) {
-          map.set(key, value);
-        } else {
-          const curr = map.get(key) || new Balance(0);
-          map.set(key, curr.add(value) as Balance);
-        }
-
-        return map;
-      }, new Map<string, Balance>())
+       list.reduce((map, { era, value }) => {
```

Comment:
This could simply be (preferred) `return list.reduce` (without the `return ()` wrapper)